### ISSUE TITLE
Question: how to take out the image path when using Pipeline?

### ISSUE THREAD

**Issue by XiaoqiangZhou** [NVIDIA/DALI#1221]
Hi, thanks for your contribution for building this pachage. Could you please help me with the following problem?

I use `ops.FileReader` to read training data & labels. But it seems the pipeline directly return the image content, I do not know **how to take the information of the image file path from the pipeline**, which is needed because I need to provide an inferenced image path in my code to save it.

Part of my code is listed below:

```
    self.input = ops.FileReader(file_root=data_dir, file_list=flist_path)
    self.jpegs, self.labels = self.input(name="Reader")
    output = self.decode(self.jpegs)
return output, self.jpegs

pip_train = HybridTrainPipe(batch_size=batch_size, num_threads=num_threads, data_dir=image_dir, flist_path = flist_path, crop=crop)
dali_iter_train = DALIGenericIterator(pip_train, output_map=["data", "path"])
```
I thought `self.jpegs` is the image path I wanted, but I was wrong. `self.jpegs` is a tensor with shape `[Batch_Size, 1230454]` and `dtype=torch.uint8.`

Could you please tell me how to extract the image path information from the pipeline?
Thanks!

**Comment by awolant** [comment]
Hello, thanks for the question.
 There is no support for this via `FileReader` op right now, but you can still accomplish it using [ExternalSource](https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/examples/external_input.html). Exact info on how to do this you can find in #1130 

FYI, in this context `self.jpegs` contains non decoded jpeg data.

We also have another opened issue about this request: #1170, so let me close this one to avoid duplication.
If you have more questions or comments, please don't hesitate to reopen or post in this other issue.
